doc/source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst

```diff
@@ -4,8 +4,6 @@
  XtraDB Performance Improvements for I/O-Bound Highly-Concurrent Workloads
 ===============================================================================
 
-In |Percona Server| :rn:`5.6.13-61.0` a number of |XtraDB| performance improvements have been implemented for high-concurrency scenarios.
-
 Priority refill for the buffer pool free list 
```

Comment:
Not sure why github is still showing this but I removed the version from that sentence (View full changes shows correct diff).